src/components/Search/SearchDropdown.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import FormControl from '@mui/material/FormControl';
+import FormHelperText from '@mui/material/FormHelperText';
+import Grid from '@mui/material/Grid';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import SearchViewHelper from '../../utils/searchViewHelper';
+
+const { handleChangeDropdown } = SearchViewHelper;
+
+function SearchInput(props) {
+  const { dropdownError, dropdownHelperText, repository } = props;
+  const repoList = ['try', 'autoland', 'mozilla-central'];
```

Comment:
I wonder if this list could be useful in a central file, to be reused in other places. This is more a passing-by comment and can wait for a future patch until we need it.

Reply:
I guess that now you exported this in a constant file in the other PR, you can import it here after the other PR lands.

I'm fine if you'd rather do it in another small PR too.